.github/workflows/claude-octopus.yml

```
@@ -124,6 +116,7 @@ jobs:
     permissions:
       contents: read
       pull-requests: write
+      copilot-requests: write
```

actionlint (1.7.12)

[error] unknown permission scope "copilot-requests". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)